Add osx_arm64 and linux_aarch64

rfiorella opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #12 

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

rfiorella committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.16.22.08.15

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

rfiorella committed:
add osx_arm64 and linux_aarch64

rfiorella committed:
add make to build

rfiorella committed:
bump build number

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ Use of `posix` package on windows is deprecated. Use `m2-base`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19416189372](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19416189372). Examine the logs at this URL for more detail.</sub>

rfiorella committed:
fix linter suggestion

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

rfiorella committed:
Add missing win utilities

Linter suggests moving to m2-base from deprecated posix,
but posix has a lot of utilities not in m2-base. Adding these
back in.

rfiorella commented:
@jakirkham I think this one is ready to go!

rfiorella commented:
@jakirkham hate to be a bother, but you could take a look? This is holding up a build of subversion-feedstock. Thanks!

rfiorella commented:
@conda-forge/core mind taking a look?

h-vetinari commented:
Please add yourself to maintainers (+ rerender), and I'll be happy to merge. Thanks for fixing up this feedstock!

rfiorella committed:
update maintainers

rfiorella committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.22.22.48.19

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.0
- rattler-build-conda-compat 1.4.10

rfiorella commented:
@h-vetinari done, thanks! Looks like this would close #12, but not #11 (#11 has ppc64le though so not completely duplicated here).
